packages/mgt-components/src/components/mgt-people-picker/mgt-people-picker.ts

```diff
@@ -854,7 +854,8 @@ export class MgtPeoplePicker extends MgtTemplatedTaskComponent {
             id="${person.id}"
             class="searched-people-list-result"
             role="option"
-            @click="${() => this.handleSuggestionClick(person)}">
+            @click="${() => this.handleSuggestionClick(person)}"
+            @keydown="${(e: KeyboardEvent) => this.handleSuggestionKeydown(person, e)}">
```

Comment:
I'm confused to see this here. keyboard based selection of suggestions is working without this via the keydown handler on the list itself AFAIK. 

I'm not sure what adding this extra code is giving us here?

Reply:
Removing. Previous fix implementation but no longer necessary